product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx

```diff
@@ -71,3 +71,8 @@ Keep the following points in mind when designing a replication system:
 
 !!! Note
     For partition tables, replication is supported only when all the partition tables have a primary key.
+
+-   **Number of tables and views in publications**: 
```

Comment:
Is this and the set of text that follows meant to be a bullet uner the not about partition tables or simply a bulleted set of text like the other bulleted items on the page. In fact, I'm not sure why the text about partition tables is called out in a note rather than just another bulleted list item.  If for some reason, we feel that the partition table related text needs to be a Note, I would suggest moving this text about the number of tables an views before the note.

Reply:
Hi @mlewandowski-edb, I agree the location of the note is a bit confusing and I thought it belonged to the bullet above, hence why I left it there. But while re-reading it, I don't think the note is related to the text before it, and it seems like you feel the same. 
WDYT if I just convert the note into a "normal" bullet list item? 
Also, as I was going through this page, I thought we could make some improvements, so I created https://enterprisedb.atlassian.net/browse/DOCS-1199 to tackle at a later time.

Reply:
@mlewandowski-edb I discussed this with the team earlier. They feel that this text should remain a note, as it is important information, so I moved the note to the bottom so there is no confusion on whether the note belongs to one of the bullet items. 
However, as mentioned before, I plan to reorganize this page at some point and opened a task for this [DOCS-1199 ](https://enterprisedb.atlassian.net/browse/DOCS-1199).